iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -1154,7 +1154,17 @@ private async Task<Message> SendTwinRequestAsync(Message request, string rid, Ca
                         {
                             if (status >= 300)
                             {
-                                throw new IotHubException($"Request {rid} returned status {status}", isTransient: false);
+                                // The Hub team is refactoring the retriable status codes without breaking changes to the existing ones.
+                                // It can be expected that we may bring more retriable codes here in the future.
+                                // Retry for Http status code 429 (too many requests)
+                                if (status == 429)
```

Comment:
Can you use `HttpStatusCode.TooManyRequests` instead to improve readability?

Reply:
It appears that [net451](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=netframework-4.5.1), [net472 ](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=netframework-4.7.2)and [netstandard2.0](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=netstandard-2.0) don't include `TooManyRequests` in the `HttpStatusCode` enum which they support.

Reply:
OK